fastlane/actions/fhir_java_r4.rb

```diff
@@ -67,6 +67,17 @@ def self.includeStatics
           sh "cp #{modelSource}/MedicationStatementTest.java #{modelTestTarget}"
       end
 
+	  # Hotfix for broken generation of Device
+	  def self.cleanupDevice
+	  	  location = "./fhir-java/src-gen/main/java/care/data4life/fhir/r4/model"
+	  	  text = File.read(location + "/Device.java")
```

Comment:
What was the issue?

Reply:
In Device it leaves a line (see the RegEx), which should be at a comment. The KMP version is fine though.